.github/workflows/build.yml

```
@@ -77,21 +84,36 @@ jobs:
                   build_doc = "false"
           oses = ["macos", "ubuntu", "windows"]
           build_dict = {os : [k for k in build if k.startswith(os)] for os in oses}
-          print(f"::set-output name=build::{build_dict}")
-          print(f"::set-output name=test::{dict({os : [k for k in test if k.startswith(os)] for os in oses})}")
-          print(f"::set-output name=build_doc::{build_doc}")
-          for os in oses:
-              print(f"::set-output name=do_{os}::{'true' if len(build_dict[os]) > 0 else 'false'}")
-          print(f"::set-output name=python_version::{python_version}")
+          with open(getenv("GITHUB_OUTPUT", default="/tmp/t"), "a") as f:
+              f.write(f"build={build_dict}")
```

Add newlines everywhere:
```suggestion
              f.write(f"build={build_dict}\n")
```

---

Merci